src/Glimpse.Common/Broker/Message.cs

```diff
@@ -9,14 +9,16 @@ public class Message : IMessage
     {
         public Guid Id { get; set; }
 
-        public string Type { get; set; }
+        public IEnumerable<string> Types { get; set; }
 
+        [JsonConverter(typeof(RawStringConverter))]
         public string Payload { get; set; }
 
-        public MessageContext Context { get; set; }
+        public int Ordinal { get; set; }
 
-        public IEnumerable<string> Tags { get; set; }
+        public MessageContext Context { get; set; }
 
+        [JsonIgnore]
```

Comment:
> I wonder if we should just manually serialize this type?

Do you mean literally append together the the curly braces and such using a `StringBuilder` or something? It'd certainly be interesting to see the perf difference on that.